vm_supervisor/vm/firecracker/program.py

```diff
@@ -388,9 +390,13 @@ async def _setup_configuration(
         ipv6 = self.get_vm_ipv6()
         ipv6_gateway = self.get_vm_ipv6_gateway()
 
-        if not settings.DNS_NAMESERVERS:
+        dns_servers = settings.DNS_NAMESERVERS
```

Comment:
I think it would be great to have a `dns_servers()` method in `Settings` that uses `ipaddress` to perform the conversion to `IPv4/IPv6 address objects only once.